### PR TITLE
ci: DLT-1734 update email action to use a cloud function

### PR DESCRIPTION
**Pull request by juliodialpad**
# Update email action to use a cloud function

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1734

## :book: Description

- Updated the action to send the email through a cloud function.

## :bulb: Context

We were using an external `dialtone.updates@gmail` email account to send internal communications, that was working but It's not a good approach.

Changed to use a secure cloud function to send the emails.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [ ] I have added all relevant documentation.

## :crystal_ball: Next Steps

- Remove test files and update email account

### CODE REVIEW

**Review by iropolo — APPROVED** [review]
action looks good.

File: .github/workflows/send-blog-communications.yml [hunk 1]
```diff
@@ -67,48 +72,66 @@ jobs:
...
+          -e 's_(\[.*\])\(([^)]+)\)_\1(https://dialtone.dialpad.com\2)_g' \
+          $FILE_PATH;
+          sed -i 10q $FILE_PATH;
           echo -e "... Read the full article: $POST_URL" >> $FILE_PATH;
+          {
+            echo 'MESSAGE_BODY<<EOF'
+            cat $FILE_PATH | tr -d '\n'
+            echo
+            echo EOF
+          } >> $GITHUB_ENV;
```
Comment (braddialpad) [inline .github/workflows/send-blog-communications.yml:102]: Guess we gotta be careful if we ever change this structure..

**Review by braddialpad — APPROVED** [review]
Looks good, thanks.

File: .github/workflows/send-blog-communications.yml (hunk 1 above)
Comment (juliodialpad) [reply .github/workflows/send-blog-communications.yml:102]: Certainly, if this continues growing, we might want to create a script to handle this.

**Pull request merged**